src/components/widgets/SearchQueryList.vue

```diff
@@ -183,13 +196,28 @@ export default {
       toggleGroupId: null
     }
   },
+
   computed: {
+    ...mapGetters(['currentProduction', 'isCurrentUserManager']),
+
     sortedFilters() {
       return sortByName([...this.queries])
     },
+
     userFilters() {
-      return this.sortedFilters.filter(query => !query.search_filter_group_id)
+      return this.sortedFilters
+        .filter(query => !query.search_filter_group_id)
+        .sort((a, b) => {
+          if (a.is_shared && !b.is_shared) {
+            return -1
+          } else if (!a.is_shared && b.is_shared) {
+            return 1
+          } else {
+            return a.name.localeCompare(b.name)
```

Comment:
The `sortedFilters` list is already filtered by name.
returning `0` seems sufficient, right?